### DISCUSSION TITLE
How do I delete everything and start over?

### DISCUSSION THREAD

**Discussion by LiamNgo24** [photoprism/photoprism#3857] [Q&A] [answered]
I accidentally deleted the folder .../database/photoprism while solving another issue and now photoprism would not start.

I have tried "docker compose rm -s -v" and deleting the docker volumes used by mariadb and photoprism but upon reinstalling via "docker compose up -d". After googling and asking chatgpt I finally decided that I want to start all over (the photos werent important as they are backed up elsewhere).

How do I delete everything to start over?

**Comment by dwichers** [comment, marked as answer]
1: Stop Photoprism
docker-compose down
docker stop photoprism
docker rm photoprism

2: Remove Docker containers and images
docker rmi photoprism/photoprism:latest

3: Remove Docker volumes
docker volume rm photoprism_photoprism-originals
docker volume rm photoprism_photoprism-storage
docker volume rm photoprism_photoprism-database
